fcrepo-kernel/src/main/java/org/fcrepo/kernel/observer/FedoraEvent.java

```diff
@@ -184,6 +192,13 @@ public long getDate() {
         }
     }
 
+    /**
+     * Get the event ID.
+    **/
+    public String getEventID() {
```

Comment:
I realized I was confusing getIdentifier() with getPath() (the output of getPath() is used in the identifier JMS header...).  So I'll go ahead and remove the getIdentifier method, and rework the handful of tests that use it.